analytical_engine/CMakeLists.txt

```diff
@@ -374,6 +374,22 @@ add_custom_target(gsa_clformat
     COMMENT "Running clang-format, using clang-format-8 from https://github.com/muttleyxd/clang-tools-static-binaries/releases"
     VERBATIM)
 
```

Comment:
@sighingnow Hi, could you come up with any better idea for building maven project inside CMake?